packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -21,11 +21,11 @@ import useLockBodyScroll from '@theme/hooks/useLockBodyScroll';
 
 import styles from './styles.module.css';
 
-function NavLink({activeBaseRoute, to, href, label, position, ...props}) {
+function NavLink({activeBasePath, to, href, label, position, ...props}) {
   const toUrl = useBaseUrl(to);
-  const activeBaseUrl = useBaseUrl(activeBaseRoute);
+  const activeBaseUrl = useBaseUrl(activeBasePath);
   const activeBaseProps =
-    activeBaseRoute != null
+    activeBasePath != null
```

Comment:
I wonder why not use strict comparison here?

Reply:
Because I want to check both `null` and `undefined`. `!= null` checks for both.

Reply:
I would then remove this check altogether, but ok.

Reply:
I guess I can do that. The other falsey values aren't valid path values anyway.